src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -105,7 +105,7 @@
       IsReadOnly="{Binding IsProjectPackageReference, Converter={StaticResource NotNullOrTrueToBooleanConverter}}"
       StaysOpenOnEdit="{Binding IsProjectPackageReference, Converter={StaticResource InverseNullToVisibilityConverter}}"
       ItemsSource="{Binding Path=VersionsView}"
-      Text="{Binding Path=UserInput, Mode=OneWay}"
```

Comment:
Making it two way so when we change the `SelectedVersion` we can set the `UserInput` and trigger a `Versions.Rerfesh()` and the filter activates

Reply:
Based on our sync earlier, we removed the `Refresh()`, so this can be reverted?